croutonjs/src/templates/meetings.hbs

```diff
@@ -42,11 +42,7 @@
 						  role="button"></span>{{ this.formats }}
 				</a>
 			{{/if}}
-			{{#if this.is_virtual}}
-				<div></div>
```

Comment:
What is this old `is_virtual` if condition for, why is it safe to remove, and what is stored in `formatted_comments`?

Reply:
The old `is_virtual` is code that I forgot to clean up in prior versions.  It was a boolean that basically checked if the root server was `bmlt.virtual-na.org`.   That is no longer the case since we have the format codes now to make that determination.

Reply:
I am researching the 2nd part to your question.